functions-methods.js

```diff
@@ -33,4 +52,15 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+
+function checkEmailValidity (emailAdress){
+    if (emailAdress.includes('@')  && (emailAdress.slice(-1) !== '.')
+        && (emailAdress.includes(',') === false))
+        return true;
+    else
+        return false;
```

Comment:
Dit kun je korter schrijven
```suggestion
    return emailAdress.includes('@') && emailAdress.slice(-1) !== '.' && !emailAdress.includes(','));
```